packages/bsky/src/data-plane/server/routes/subscription-entitlements.ts

```diff
@@ -3,6 +3,18 @@ import { Service } from '../../../proto/bsky_connect'
 import { Database } from '../db'
 
 export default (db: Database): Partial<ServiceImpl<typeof Service>> => ({
+  async getSubscriptionEntitlement(req) {
+    const { dids } = req
+
+    return {
+      subscriptionEntitlements: await db.db
```

Comment:
we want to make sure these are returned in the same order as the params

and that the response as `undefined` as the array item that corresponds to each param that does not have entitlements

you can find similar logic in other dataplane routes such as in the `relationships` file